sds_data_manager/lambda_code/SDSCode/pipeline_lambdas/dependency.py

```diff
@@ -819,6 +819,8 @@ def lambda_handler(event, context):
     else:
         dependencies_output = json.dumps(dependencies)
 
+    logger.info(f"Found dependencies: {dependencies}.")
```

Comment:
Do you want to add what they were for to make sure this log is traceable to the trigger?